src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotificationMediationUnsuccessfulClaimantLRHandler.java

```diff
@@ -110,19 +111,13 @@ private void sendEmail(final CaseData caseData) {
                 String.format(LOG_MEDIATION_UNSUCCESSFUL_CLAIMANT_LR, caseData.getLegacyCaseReference())
             );
         }
-
     }
 
     private void sendMailAccordingToReason(CaseData caseData) {
-        if (findMediationUnsuccessfulReason(caseData, List.of(NOT_CONTACTABLE_CLAIMANT_ONE))) {
+        if (findMediationUnsuccessfulReason(caseData, List.of(NOT_CONTACTABLE_CLAIMANT_ONE, NOT_CONTACTABLE_CLAIMANT_TWO))) {
             sendMailRepresentedClaimantNoAttendance(caseData);
-        } else if (findMediationUnsuccessfulReason(caseData, List.of(
```

Comment:
the reason why this bit of code is removed is because:
1. this piece of work was spread out to different tickets therefore the whole picture wasn't captured. In the end, we only needed to pay attention to `NOT_CONTACTABLE_CLAIMANT_ONE` and `NOT_CONTACTABLE_CLAIMANT_TWO`, as the rest of the reasons gets the same email
2. no matter the reason, LIPs get the same email to say "something has been updated"